pkg/operator/e2e/main_test.go

```diff
@@ -340,7 +340,7 @@ func testRuleEvaluatorDeployment(ctx context.Context, t *testContext) {
 		// When not using GCM, we check the available replicas rather than ready ones
 		// as the rule-evaluator's readyness probe does check for connectivity to GCM.
 		if skipGCM {
-			if *deploy.Spec.Replicas != deploy.Status.AvailableReplicas {
+			if *deploy.Spec.Replicas != deploy.Status.UpdatedReplicas {
```

Comment:
Oh, I misunderstood ready vs available.

What we maybe could do at some point is spin up a local CreateTimeSeries endpoint (takes maybe 20 LOC) and point kind tests at that one.

Reply:
Added a TODO.